### PR TITLE
Feat/admin/custom products type

### PR DESCRIPTION
**Pull request by nguyenthanhtuanIT**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
+- `Copy` fordel `src`, then `paste` in fordel `Product`.
+- example: `packages/GGPHP/Product/src`
 ### Step-3
 - Now, to register the service provider, go to the `app.php` file inside the config folder & add your service provider inside the `providers` array.
-- `GGPHP\Admin\Providers\AdminServiceProvider::class`
+- `GGPHP\Product\Providers\ProductServiceProvider::class`
 ### Step-4
-- we need to add our package to the `composer.json` file of project root for auto loading in `psr-4`.
-- ` "GGPHP\\Admin\\": "packages/GGPHP/Admin/src"`
+- we need to add our package to the `composecustomr.json` file of project root for auto loading in `psr-4`.
```
Comment (HuynhHungManh) [inline README.md]: composecustomr là gì tuấn ?

File: README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
 # bagisto-products
-Using bagisto products
+## Features
+- Admin able custom list all products or show parent products only
```
Comment (HuynhHungManh) [inline README.md]: `list all products` => `all products` or `product listing`

File: README.md [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+- Admin able custom configuration product types suiable with your project
```
Comment (HuynhHungManh) [inline README.md]: `custom configuration` => `config`

File: README.md [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
+
+## Using bagisto products
 
 ### Step-1
 - Download(clone) source code.
 ### Step-2
-- `Copy` fordel `src`, then `paste` in your fordel package.
-- example: `packages/GGPHP/Admin/src` (`GGPHP` is your company name or namespace, `Admin` ís your package name, `src` is fordel copied)
+- Inside fordel `packages` create new fordel `GGPHP`, inside fordel `GGPHP` create fordel `Product`
```
Comment (HuynhHungManh) [inline README.md]: `fordel` => `folder`

Comment (nguyenthanhtuanIT) [reply README.md]: fixed

**Pull request merged**